rust/azure_iot_operations_mqtt/src/lib.rs

```diff
@@ -28,6 +28,8 @@ pub mod interface_mocks;
 
 #[macro_use]
 extern crate derive_builder;
+#[macro_use]
+extern crate derive_getters;
```

Comment:
Was this actually necessary to add? I didn't see anything in the docs that required this declaration, so I'm assuming you did this because we did it for `derive_builder` which may not apply here. Although if this is required, this is fine.

Reply:
Discussed offline; for posterity: added for consistency with `derive_builder`, and it adds the derive macro into the global namespace so it doesn't have to be `use`d repeatedly if we want to use it more in the future.

Reply:
This explains why I thought I was going crazy about sometimes needing to do `use` for derive_builder and sometimes not needing to 😅